src/PictureCard/index.js.flow

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import type { Globals } from "../common/common.js.flow";
+
+type Image = {|
+  +original: string,
+  +placeholder: string,
+  +name: string,
+|};
+
+export type Props = {|
+  ...Globals,
+  +actions?: React$Node,
+  +children: React$Node,
+  +label?: React$Node,
+  +title: React$Node,
```

Comment:
All properties should be optional, because there is just a lot of variants when comes to usage.